### ISSUE TITLE
Unable to Set Up and Run Edge Impulse Mobile Client Locally for Modification

### ISSUE THREAD

**Issue by NEHAL608** [edgeimpulse/mobile-client#17]
Hi, I’m new to Node.js development and primarily work in embedded systems. I'm trying to run the **Edge Impulse Mobile Client** on my local machine so I can modify it to fit the requirements of my custom application. However, I’m facing difficulties setting it up and getting it to run. I would really appreciate some guidance on how to resolve these issues.

Steps Taken:
Here’s what I’ve done so far:

1. - Cloned the mobile-client repository to my local machine.
2. - Installed all the dependencies by running npm install in the project root.
3. -Tried running the project using npm run start, but I encountered errors.

Issues:
Module not found:
When I try to run the code, it fails to find the module @ei/common and other related imports. To resolve this, I manually created a common folder, but the problem persists.
TypeScript compilation errors:
After creating the common folder, I ran into issues with TypeScript compilation. Specifically, I get the following error:

**TS18003: No inputs were found in config file 'tsconfig.json'**   

Could you provide a step-by-step guide on how to properly configure and run the Edge Impulse Mobile Client locally so that I can make modifications?

**Comment by mtourne** [comment]
Yes it seems like the package @ei/common is not distributed anywhere.

This is blocking to understand how to roll out my own browser code that has some edge vision capabilities.

**Comment by janjongboom** [comment]
@NEHAL608 @mtourne Apologies that it took so long; but I only saw this issue last week... This has now been fixed.
